Generator/Generator.csproj

```diff
@@ -1,68 +1,17 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{DDD005C7-D63D-4707-9464-1BCB9FD00454}</ProjectGuid>
+    <TargetFramework>net47</TargetFramework>

```

Comment:
Is there a specific reason to jump to 4.7? My machine for some reason doesn't see the 4.7 framework and it was causing problems. I switched this back to 4.5.2 and it was fine.

Reply:
no, there's nothing special about that... I usually try to target frameworks and platforms as low as possible for libraries/assemblies/packages, as opposed to frontend executables, where I try to use the highest version of the framework of the platform available.

If 4.7 is failing to you, you're probably missing some SDK of Visual Studio 2017

Reply:
> If 4.7 is failing to you, you're probably missing some SDK of Visual Studio 2017

Yeah, have been trying to figure that out. Not sure why VS doesn't want to recognize it. It's supposed to come with VS.

Reply:
I have lowered the the requirement to 4.5.2, but if you're on VS2017, you should definitely support 4.7

Reply:
Yeah, I don't know what's wrong yet. I updated my 2017 just now and I also manually installed 4.7.1. Will see if that changes anything, but I think lowering the requirement doesn't hurt.